map.kenny.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "KennySB-dev",
+        "email": "kennybugeja@gmail.com",
+        "discord": "kenny.sb"
+    },
+    "record": {
+        "CNAME": "mango.fps.ms:26926"
```

Comment:
Invalid CNAME record (you can't put the port in the CNAME)

Reply:
ah wait but i need to point at the port, how will that work?

Reply:
well, if there is no way to do it, are url records working now, or are they still broken?